simulariumio/data_objects/display_data.py

```diff
@@ -72,6 +73,25 @@ def __init__(
             raise DataError(f"{color} should be provided as '#xxxxxx' or '#xxx'")
         self.color = color
 
+    @classmethod
+    def from_dict(
+        cls,
+        display_info: Dict[str, Any],
+        default_display_type: DISPLAY_TYPE = DISPLAY_TYPE.SPHERE
+    ):
+        """
+        Create DisplayData from a simularium JSON dict containing buffers.
+        """
+        if display_info is None:
+            return cls()
+        return cls(
+            name=display_info.get("name"),
```

Comment:
should name use a default empty string?

Reply:
Yeah I'll do that. Hopefully this isn't a case we run into, since name is a required field for display data, but `name=""` does feel better than `name=None`